crates/pipeline_manager/src/api.rs

```diff
@@ -1742,6 +1742,8 @@ pub struct PipelineIdOrNameQuery {
         ("pipeline_id" = Uuid, Path, description = "Unique pipeline identifier."),
         ("table_name" = String, Path, description = "SQL table name."),
         ("format" = String, Query, description = "Input data format, e.g., 'csv' or 'json'."),
+        ("array" = Option<bool>, Query, description = "Set to `true` if updates in this stream are packaged into JSON arrays (used in conjunction with `format=json`). The default values is `false`."),
+        ("update_format" = Option<JsonUpdateFormat>, Query, description = "JSON data change event format (used in conjunction with `format=json`).  The default value is 'insert_delete'."),
     ),
     tag = "Pipelines",
     request_body(
```

Comment:
probably needs another request_body for json?

Reply:
Unfortunately, utoipa doesn't support this yet: https://github.com/juhaku/utoipa/issues/692